messaging.rst

```diff
@@ -4,7 +4,36 @@ Raiden Messages Specification
 Overview
 ========
 
-This is the specification document for the messages used in the Raiden protocol.
+This is the specification of the messages used in the Raiden protocol.
```

Comment:
Can yu maybe add something like. There are x different messages in raiden. we use y for the coordination of a mediated transfer and z for the channel withdraw

Reply:
@karlb suggested a different classification of the messages that will be in the followup pr.

Reply:
ah ok, then ignore my comment.